.browserslistrc

```diff
@@ -0,0 +1,9 @@
+# See our browser support policy at https://go.czi.team/edu-browser-support.
```

Comment:
FYI, I'm updating this in https://github.com/FB-PLP/traject/pull/9663

Reply:
Oh wait, this diff is probably because this branch is out of date with `next` or something. Never mind.